### PR TITLE
MERGE LIGHT THEME:

### PR DESCRIPTION
**Pull request by Keeghaan**
Here's an idea of the colors I tried for our site. I hope you'll like it.
The commit config script wasnt on (I really don't understand why ^^') so I hope the last commit is OK

### CODE REVIEW

**Review by AdrianWeaver — COMMENTED** [review]
Left some comments, will approve once the conversations are approved with or without fixes as these are minor concerns.

File: front/app/src/Theme/CustomLightTheme.ts
Comment (Keeghaan) [reply front/app/src/Theme/CustomLightTheme.ts:21]: I forgot to remove this comment :x I do it right now

File: front/app/src/Component/MenuBar/IconTitle.tsx
Comment (Keeghaan) [reply front/app/src/Component/MenuBar/IconTitle.tsx:8]: fontConfig and icon are default import from their file (they could be named otherwise, they would work as the default export you can find in their file)
Icon is not default, it will work as itself (The export has the same name)
Sorry it's hard for me to explain anything at all. But it is not a mistake ^^

**Comment by Keeghaan** [comment]
Updated

**Review by AdrianWeaver — APPROVED** [review]

**Pull request merged**
